form_registration.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
By account I mean the `accounts` table - https://github.com/ka8725/monmon/blob/master/commands.sql#L2